scalafmt-tests/src/test/resources/spaces/InterpolatedStringCurlyBracesAlways.stat

```diff
@@ -0,0 +1,21 @@
+spaces.inInterpolatedStringCurlyBraces = Always
+
+<<< Simple variable
+s"The great ${white} hope"
+>>>
+s"The great ${ white } hope"
+
+<<< Complex statement
+s"The racism ${doesn.t} exist"
+>>>
+s"The racism ${ doesn.t } exist"
+
+<<< Function call
+s"Four ${more()} years!"
+>>>
+s"Four ${ more() } years!"
```

Comment:
This is sort of a drive-by comment since I happened to look at this pr. While I'm not a maintainer of scalafmt, I feel like I'm within my right as a member of the scalameta org to ask that this be changed.

I'm not sure if this is an attempt at humor or something else, but given the implications of what you're saying here, jokingly or not, is uncalled for and inappropriate. Please change this.

cc @gabro @olafurpg

Reply:
\* *boo-hoo* \*

…just kidding :)

Reply:
I fully support what @ckipp01 said 👍 

Thanks @arbitrary-dev for changing addressing Chris's comment.

Reply:
@ckipp01 is right. we should avoid any controversy in open source. i have deleted your reply as not adding to world peace.

Reply:
Thank you @ckipp01 for bringing this up. I 100% agree with you.